### DISCUSSION TITLE
Welcome to tract Discussions!

### DISCUSSION THREAD

**Discussion by kali** [sonos/tract#419] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zicklag** [comment]
Hi! I think the concept behind this tool is great. With it, regardless of how I have designed my neural network, I can run it anywhere I can run Rust! I've just barely gotten into machine learning, but it has interested me for a while and I think this will be useful to me when I find the time to experiment. :smiley:
